js/src/wp-seo-post-scraper.js

```diff
@@ -1,9 +1,13 @@
-/* global YoastSEO, tinyMCE, wp, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console */
+/* global YoastSEO, tinyMCE, ajaxurl, wpseoPostScraperL10n, YoastShortcodePlugin, YoastReplaceVarPlugin, console, require */
 (function( $ ) {
 	'use strict';
 
 	var currentKeyword = '';
 
+	// Initialize the keyword tab module.
+	var keywordTab;
```

Comment:
Yes that's right. 

In this case I think it better to not follow your advice, since it's a convention in JS to write modules/prototypes with a capital K and start variable names with a lower k. @atimmer What do you think?